end_to_end_tests/golden-record/my_test_api_client/models/__init__.py

```diff
@@ -27,5 +27,5 @@
 from .model_with_union_property_inlined_fruit_type0 import ModelWithUnionPropertyInlinedFruitType0
 from .model_with_union_property_inlined_fruit_type1 import ModelWithUnionPropertyInlinedFruitType1
 from .test_inline_objects_json_body import TestInlineObjectsJsonBody
-from .test_inline_objects_response_200 import TestInlineObjectsResponse_200
+from .test_inline_objects_response200 import TestInlineObjectsResponse200
```

Comment:
Another issue 😞. I imagine this is why all of the existing string case libraries get it wrong... so many edge cases. I'll add a unit test for this one.

Reply:
Yep, that's exactly what I was talking about in my [previous comment](https://github.com/triaxtec/openapi-python-client/pull/375#issuecomment-812098844).